database/database.sql

```diff
@@ -0,0 +1,42 @@
+CREATE DATABASE IF NOT EXISTS team_server;
+
+CREATE TABLE IF NOT EXISTS players (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Should we make primary keys ints or uids? Which DB are we using MySQL or PostgreSQL?

Reply:
I have the most experience with MySQL. So MySQL unless you prefer Postgre